README.md

```diff
@@ -1 +1,85 @@
-# book-dirt-bike-front-end
\ No newline at end of file
+# Dirt Bikes
+
```

Comment:
- [ ] When creating a new bike, this happens:

![image](https://user-images.githubusercontent.com/57421823/193099908-f3f78584-582e-426c-88ef-4bb6ac14cb39.png)

The bike is (very sloooowly) created, but a error message appears that it was not.

Kindly squash this bug, so your project works as intended.

Reply:
Thanks for your your review @Meltrust. We have fixed it.